ethereum/p2p/src/test/java/tech/pegasys/pantheon/ethereum/p2p/discovery/internal/PeerDiscoveryControllerTest.java

```diff
@@ -228,28 +228,40 @@ public void bootstrapPeersPongReceived_HashMatched() {
         .hasSize(3);
 
     // Simulate a PONG message from peer 0.
-    final PongPacketData packetData =
-        PongPacketData.create(localPeer.getEndpoint(), mockPacket.getHash());
-    final Packet packet = Packet.create(PacketType.PONG, packetData, keyPairs.get(0));
-    controller.onMessage(packet, peers.get(0));
+
+    for (int i = 0; i < 3; i++) {
```

Comment:
```suggestion
    for (int i = 0; i < peers.size(); i++) {
```

Reply:
Actually - I'm wondering why you changed this test to have all peers responding instead of just one?

Reply:
the "bonding round" needs to conclude before the "find neighbours round" can begin. one way to achieve that is for all peers to respond, the other way is for the round to timeout.